src/components/sideBarContentEdit/SidebarContentEdit.tsx

```diff
@@ -130,7 +130,8 @@ const SidebarContentEdit = ({
   };
   const updateTags = (tags: string[]) => {
     const updatedTranscript = getUpdatedTranscript();
-    updatedTranscript.tags = tags;
+    const lowerCaseTags = tags.map((tag) => tag.toLowerCase());
+    updatedTranscript.tags = lowerCaseTags;
```

Comment:
It seems that the logic introduced with 069bef7795a089f3e7738d4f6ecfe36064ae0707 does not solve the problem with the capitalized tags. Looking at the code, we might think that we change the state (in this case `tags`) but in reality, when you click the Submit button, it access a different state which is what it updates the PR with.

This is my understanding of the issue: 
When in the Editor, the state of the metadata (which `tags` is part of) is managed through `sideBarData` which is also accessed through `getUpdatedContent()` and mutated through `sideBarContentUpdater()` which is passed as `updater` to the nested sidebar component in order to be able to change the state of the metadata through the sidebar selection. 

The issue with this new change is that when  `tags` are updated through the dropdown, the updated lowercase value is send to the backend using `saveTranscript()` but `updater()` updates the state with the non-lowercase `tags`. So when `handleSubmit()` reads the state before creating the PR, it reads the state with the non-lowercase `tags`. 

You could just also use the `lowerCaseTags` for the `updater()` but then tags are presented to the user as lowercase which is not what we want. I think that a better solution would be to transform tags to lowercase as closer to the PR-creation as possible, which in this case seems to be here:
https://github.com/bitcointranscripts/transcription-review-front-end/blob/092ad3848e7ce344a2c076957129208ea9710371/src/components/transcript/index.tsx#L221

Reply:
We can keep tags always as lowercase, from when we get the data and when we update the data as you have mentioned.

For the user a simple CSS `text-transform: capitalize` will do the capitalization, so tags are always lowercase in our logic.